cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/Differ.java

```diff
@@ -87,16 +90,22 @@ public DiffNode createDiffTree(Bean liveConfig, Bean desiredConfig) {
     private DiffNode doCreateDiffTree(Bean liveConfig, Bean desiredConfig) {
         Diff diff = JAVERS.compare(liveConfig, desiredConfig);
 
+        log.verbose("Parsing JaVers change objects to custom change objects");
         // parse the change objects created by the JaVers diff to custom change objects
         List<CfChange> cfChanges = diff.getChanges()
                 .stream()
+                .peek(change -> log.verbose("JaVers found a change:", change))
```

Comment:
We should consider to use `debug` logging here because when there are many changes, the console output will be quite long and confusing